core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -172,10 +172,8 @@ class SparkHadoopUtil extends Logging {
     }
   }
 
-  private def getFileSystemThreadStatistics(path: Path, conf: Configuration): Seq[AnyRef] = {
-    val qualifiedPath = path.getFileSystem(conf).makeQualified(path)
-    val scheme = qualifiedPath.toUri().getScheme()
-    val stats = FileSystem.getAllStatistics().filter(_.getScheme().equals(scheme))
+  private def getFileSystemThreadStatistics(conf: Configuration): Seq[AnyRef] = {
```

Comment:
does this need to take a +conf+ object anymore... can we just remove the +conf+'s throughout this callstack?